pytorch_toolkit/ote/readme.md

```diff
@@ -0,0 +1,198 @@
+# Model Templates
+
+Model Templates defines training procedure and its interface for a given neural network topology.
+
+## Directory structure
+
+A single Model Template should consist of a Python script related to model training as well
+as YAML file that will define template interface. Each model template is related to 4 scripts: `train.py`, `eval.py`, `export.py` and `quantize.py`. Model Templates may be placed in nested directories in the whole repository and will be detected
+automatically by the Platform. Directories does not need to conform to any convention as in the example below:
```

Comment:
Don't quite get the idea. Do you provide an example how directories structure does not conform to any convention?

Reply:
rephrased